vllm/v1/core/sched/scheduler.py

```diff
@@ -254,31 +254,30 @@ def schedule(self) -> SchedulerOutput:
 
                 request = self.waiting[0]
 
+                # Waiting request skipping logic
+                skip_waiting_request = False
```

Comment:
I feel we could just call it `skip_request` because this is a per request flag.
```suggestion
                skip_request = False
```

Reply:
I was debating naming this. What pushed me over is that not all requests in `self.waiting` is guaranteed to have their status `WAITING`. I see that the preempted requests also participate in this. I wanted to avoid this variable being used incorrectly in the future. But maybe I am nitpicking and `skip_waiting_request` does look cumbersome. 

I can make the change if you still feel `skip_request` is fine 👍

Reply:
Oh I didn't mean to change skipped_waiting_request"s". I'm just taking about this variable (bool) used in this loop. I'm fine with the "skipped_waiting_requests" deque.

Reply:
What about `is_skipped = False`? I actually found the variable name confusing when I first looked through the code.

Reply:
I'm ok with it.

Reply:
Updated to `is_skipped` 👍